deploy/personal/terraform.tfvars

```diff
@@ -0,0 +1,6 @@
+default_region      = "us-east-1"
```

Comment:
Should we add placeholder values for these vars rather than your details? e.g. 
`github_username     = "<YOUR_GITHUB_USERNAME>"`

Reply:
Good shout. Will fix for all variable placeholders.

Reply:
also, i like the example in the README, but let's make it generic too.
```
default_region      = "us-east-1"
docker_username     = "docker_user_123"
github_username     = "github_user_123"
github_project_name = "quickstart-react-material-ui"
app_name            = "react-material-ui-app"
environment         = "staging"
```

Reply:
Updated `deploy/personal/README.md`